src/mcp/resources/guides/init_rtdb.ts

```diff
@@ -0,0 +1,32 @@
+import { resource } from "../../resource";
+
+export const init_rtdb = resource(
```

Comment:
Do we need to use underscore case here? Seems simpler to keep camel case like the rest of the CLI.

Reply:
I was on the fence - consistent with rest of CLI vs. consistent with the other subdirectories immediately adjacent to this one. I felt like the latter was better, but can change it back if you prefer.

Reply:
Don't care too strongly either way, just noticed that it was triggering the linter.

Reply:
I fixed the linter, let's keep it this way for now.